test/parallel/test-tls-two-cas-one-string.js

```diff
@@ -1,35 +1,50 @@
 'use strict';
 
+const assert = require('assert');
 const common = require('../common');
 const tls = require('tls');
 const fs = require('fs');
 
 const ca1 =
-    fs.readFileSync(`${common.fixturesDir}/keys/ca1-cert.pem`, 'utf8');
+  fs.readFileSync(`${common.fixturesDir}/keys/ca1-cert.pem`, 'utf8');
 const ca2 =
-    fs.readFileSync(`${common.fixturesDir}/keys/ca2-cert.pem`, 'utf8');
+  fs.readFileSync(`${common.fixturesDir}/keys/ca2-cert.pem`, 'utf8');
 const cert =
-    fs.readFileSync(`${common.fixturesDir}/keys/agent3-cert.pem`, 'utf8');
+  fs.readFileSync(`${common.fixturesDir}/keys/agent3-cert.pem`, 'utf8');
 const key =
-    fs.readFileSync(`${common.fixturesDir}/keys/agent3-key.pem`, 'utf8');
+  fs.readFileSync(`${common.fixturesDir}/keys/agent3-key.pem`, 'utf8');
 
-function test(ca, next) {
-  const server = tls.createServer({ ca, cert, key }, function(conn) {
-    this.close();
-    conn.end();
-  });
 
-  server.addContext('agent3', { ca, cert, key });
+function test(ca) {
+  return new Promise(function(resolve, reject) {
```

Comment:
Is the switch to promises necessary?  I like changes to be as minimal as possible; it makes for easier reviewing and code archeology.